scripts/docker/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM jupyter/base-notebook:python-3.11.6
+
+LABEL name="pymc-marketing"
+LABEL description="Environment for PyMC Marketing"
+
+ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
+
+# Switch to jovyan to avoid container runs as root
+USER $NB_UID
+
+# Install conda environment
+COPY /scripts/docker/environment-dev.yml .
+RUN mamba env create -f environment-dev.yml && \
```

Comment:
oh does base-notebook now come with mamba? that's awesome

Reply:
Not just Mamba but also Micromamba. Guess [who](https://github.com/jupyter/docker-stacks/pull/1594) :stuck_out_tongue:

In this Docker image it's much better to use `mamba install -f environment-dev.yml` and install into the `base` environment instead of making a `pymc-marketing-dev` environment. Otherwise you're wasting lots of space by reinstalling packages that are already in the base environment of `jupyter/base-notebook`. It also makes a bunch of the stuff below unnecessary.

Reply:
Thanks @maresb ! Can you please give me a hint on how to install it in base please 🙏?

Reply:
Sure, see #543

Reply:
Amazing! Thanks!